packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -159,6 +159,20 @@ void main() {
     expect(renderBox.size.width, 80.0);
   });
 
+  testWidgets('No destinations', (WidgetTester tester) async {
```

Comment:
Could you also add this test when useMaterial3 is false? Each kind of case should be tested for both `useMaterial3` is true and false, and the other one should be added to the `group('Material 2', () {...}`.

Reply:
Updated in 00acf20b504b192394e :)